accepted/2023/wasm-browser-threads.md

```diff
@@ -0,0 +1,268 @@
+# Multi-threading on a browser
+
+## Goals
+ - CPU intensive workloads on dotnet thread pool
+ - enable blocking .Wait APIs from C# user code on all threads
+     - Current public API throws PNSE for it
+     - This is core part on MT value proposition.
+     - If people want to use existing MT code-bases, most of the time, the code is full of locks. 
+     - People want to use existing desktop/server multi-threaded code as is.
+ - allow HTTP and WS C# APIs to be used from any thread
```

Comment:
this is currently true, should we rephrase this goal to capture the new property we want? i.e. allow any HTTP/WS object to be used from any thread despite affinity

Reply:
updated